src/highdicom/sr/sop.py

```diff
@@ -164,6 +165,8 @@ def __init__(
             observer_item.VerifyingObserverName = verifying_observer_name
             observer_item.VerifyingOrganization = verifying_organization
             observer_item.VerificationDateTime = DT(now)
+            observer_item.VerifyingObserverIdentificationCodeSequence = \
```

Comment:
Is there an advantage of using `pydicom.sequence` over `[]`?

```python
            observer_item.VerifyingObserverIdentificationCodeSequence = []
```

Reply:
I don't think so, have changed as suggested